classes/install/Upgrade.inc.php

```diff
@@ -796,29 +797,30 @@ function migrateSubmissionCoverImages() {
 			);
 
 			// Create a submission_settings entry for each locale
-			foreach ($contexts[$row['context_id']]->getSupportedFormLocales() as $localeKey) {
-				$newCoverPathInfo = pathinfo($newCoverPath);
-				$submissionDao = DAORegistry::getDAO('SubmissionDAO'); /* @var $submissionDao SubmissionDAO */
-				$submissionDao->update(
-					'INSERT INTO submission_settings
+			if(isset($context)) {
```

Comment:
Was this possibly a left-over database entry for a cover image whose context (press) had been deleted? I don't think there's a legitimate case where the context shouldn't be available.

Reply:
@asmecher 
yes, this was from an old  database, where it was left-over.
I don't think, it also occurs in a real use case. Should I revert it ?

Reply:
It doesn't do any harm; I'm OK to leave it, just wondering where it was from.

Reply:
sure, then let us leave it.